keel-core/src/main/kotlin/com/netflix/spinnaker/keel/pause/Pause.kt

```diff
@@ -0,0 +1,18 @@
+package com.netflix.spinnaker.keel.pause
+
+import java.time.Instant
+
+/**
+ * A pause in an application or resource's actuation, requested by a user.
+ */
+data class Pause(
```

Comment:
`PausedRepository` now has a corresponding model to work with, like other repositories.